### ISSUE TITLE
Migrating profile pages

### ISSUE THREAD

**Issue by apple502j** [LLK/scratch-www#3853]
Migrating profile pages from scratchr2 to scratch-www is a big task, and it may be hard to do so. However, there are some good points about it:
- This is a common suggestion found on the forums.
- This may help with some slowness caused by legacy stuff.
- **This is a big one**: Currently, the API does not give certain information about the user (for example, new scratcher status or account deleted status(404)), so **3rd-party programs fetch this information from the server-side-rendered profile page. This may cause some unnecessary lags.** Migrating profile means profile API has to include all information the client needs to see - including New Scratcher status or featured project. It should return 404 for deleted ones too.

**Comment by Explosion-Scratch** [comment]
Yes, I remember #3 being mentioned on my suggestion here: scratch.mit.edu/discuss/topic/399725 this is a big problemand should be fixed.

**Comment by GrahamSH-LLK** [comment]
I agree with this suggestion. Some profile pages are taking forever to load.

**Comment by GrahamSH-LLK** [comment]
@benjiwheeler I understand that this will eventually have to happen. When are you guys planning to do this?  I think this would be well received by the scratch community.
